fixed stub/added solution/+link to open terminal

AlexandrosGazis opened the pull request:
I have provided the solution to the exercise of the curriculum.

For instructions on how to open up the Js console in our web browsers I have provided a link to the FCC's github site (#34251) and I am awaiting the pull request to be accepted . Even if it doesn't get accepted and the link remains a stub leave the link here (even if it is referring to a stub) as in time it will get completed too 👍 

Certifications/JavaScript Algorithms and Data Structures/Debugging/Use the JavaScript Console to Check the Value of a Variable

<!-- Please follow this checklist and put an x in each of the boxes, like this: [x]. It will ensure that our team takes your pull request seriously. -->

- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] My article does not contain shortened URLs or affiliate links.

If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.

Closes #XXXXX

AlexandrosGazis committed:
fixed stub/added solution/+link to open terminal

I have provided the solution to the exercise of the curriculum.

For instructions on how to open up the Js console in our web browsers I have provided a link to the FCC's github site (#34251) and I am awaiting the pull request to be accepted . Even if it doesn't get accepted and the link remains a stub leave the link here (even if it is referring to a stub) as in time it will get completed too 👍 

Certifications/JavaScript Algorithms and Data Structures/Debugging/Use the JavaScript Console to Check the Value of a Variable

lynxlynxlynx commented on guide/english/certifications/javascript-algorithms-and-data-structures/debugging/use-the-javascript-console-to-check-the-value-of-a-variable/index.md:
```diff
@@ -3,8 +3,20 @@ title: Use the JavaScript Console to Check the Value of a Variable
-<a href='https://github.com/freecodecamp/guides/blob/master/README.md' target='_blank' rel='nofollow'>This quick style guide will help ensure your pull request gets accepted</a>.
+## Hint 1:
+In order to see what is the e.g. stored value of a variable we can use the command *console.log()* .
+In this exercise it is required to use this command in order to display in our console the value of variable a so... How may I log into my console the value of a varible?
```
```suggestion
In this exercise it is required to use this command in order to display in our console the value of variable a so... How may I log into my console the value of `a` varible?
```

lynxlynxlynx commented on guide/english/certifications/javascript-algorithms-and-data-structures/debugging/use-the-javascript-console-to-check-the-value-of-a-variable/index.md:
```diff
@@ -3,8 +3,20 @@ title: Use the JavaScript Console to Check the Value of a Variable
 
-<a href='https://github.com/freecodecamp/guides/blob/master/README.md' target='_blank' rel='nofollow'>This quick style guide will help ensure your pull request gets accepted</a>.
+## Hint 1:
+In order to see what is the e.g. stored value of a variable we can use the command *console.log()* .
```
```suggestion
In order to see what is for example the stored value of a variable, we can use the command `console.log()`.
```

AlexandreLeFou replied:
ok

lynxlynxlynx commented on guide/english/certifications/javascript-algorithms-and-data-structures/debugging/use-the-javascript-console-to-check-the-value-of-a-variable/index.md:
```diff
@@ -3,8 +3,20 @@ title: Use the JavaScript Console to Check the Value of a Variable
 ## Use the JavaScript Console to Check the Value of a Variable
 
-This is a stub. <a href='https://github.com/freecodecamp/guides/tree/master/src/pages/certifications/javascript-algorithms-and-data-structures/debugging/use-the-javascript-console-to-check-the-value-of-a-variable/index.md' target='_blank' rel='nofollow'>Help our community expand it</a>.
+In order to open up the Javascript Console of your Browser (chrome,firefox,safari,opera,edge) press: [here](https://github.com/freeCodeCamp/guide/blob/master/src/pages/certifications/javascript-algorithms-and-data-structures/debugging/understanding-the-differences-between-the-freecodecamp-and-browser-console/index.md)
```
First, there are missing spaces in the browser list. More importantly, the link is pointing to the wrong URL. Even more importantly, it's not pointing to anything helpful.

AlexandrosGazis committed:
Fixed broken link+added basic command syntax

lynxlynxlynx commented on guide/english/certifications/javascript-algorithms-and-data-structures/debugging/use-the-javascript-console-to-check-the-value-of-a-variable/index.md:
```diff
@@ -3,8 +3,23 @@ title: Use the JavaScript Console to Check the Value of a Variable
 ## Use the JavaScript Console to Check the Value of a Variable
 
-This is a stub. <a href='https://github.com/freecodecamp/guides/tree/master/src/pages/certifications/javascript-algorithms-and-data-structures/debugging/use-the-javascript-console-to-check-the-value-of-a-variable/index.md' target='_blank' rel='nofollow'>Help our community expand it</a>.
+In order to open up the Javascript Console of your Browser (chrome,firefox,safari,opera,edge) press: [here](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/guide/english/certifications/javascript-algorithms-and-data-structures/debugging/understanding-the-differences-between-the-freecodecamp-and-browser-console/index.md)
```
this is still pointing to the github url instead of the live one.

AlexandrosGazis committed:
fixed broken link

changed it so as to point to live link of FCC and not the the github url(git pull request was accepted/merged: 34251)

AlexandrosGazis reviewed (commented):
got the FCC link/get a hint link

RandellDawson commented:
@lynxlynxlynx There is nothing in the Guide article to which is being linked.  If there was a link to the actual curriculum challenge, then it would be linking to a challenge which appears after this challenge.  It seems odd to reference something which will be covered in the next challenge anyway.

lynxlynxlynx commented:
I'm not sure what you're talking about, but I'm happy enough with the current state of the PR.

RandellDawson commented:
@lynxlynxlynx The sentence with the link to another guide article also does to make sense.  The OP's suggestion:

```
In order to open up the Javascript Console of your Browser (chrome,firefox,safari,opera,edge) press: [here](https://guide.freecodecamp.org/certifications/javascript-algorithms-and-data-structures/debugging/understanding-the-differences-between-the-freecodecamp-and-browser-console)
```
implies clicking the link `here` will actually open up the Javascript Console of my Browser.  If the intent of the link is to link to another article about how to open up the Javascript Console of a browser, then it would make more sense to write:
```
[Learn how to open Javascript Console of your Browser (chrome,firefox,safari,opera,edge)](https://guide.freecodecamp.org/certifications/javascript-algorithms-and-data-structures/debugging/understanding-the-differences-between-the-freecodecamp-and-browser-console)
```
However, like I said in the previous comment, it seems odd in the Guide article for the very 1st challenge in the Debugging section to be linking to a Guide article for the 2nd challenge in the Debugging section.  It would make more sense to me to have the 1st challenge (and/or Guide article for the 1st challenge)  to give this type of information instead of linking to the 2nd challenge's Guide article.  The very fact there needs to be a link to the 2nd challenge's Guide article, tells me the first challenge does not cover the right information.

lynxlynxlynx commented:
Ah, true. Still, it can be an incremental update.

ezioda004 commented on guide/english/certifications/javascript-algorithms-and-data-structures/debugging/use-the-javascript-console-to-check-the-value-of-a-variable/index.md:
```diff
@@ -3,8 +3,23 @@ title: Use the JavaScript Console to Check the Value of a Variable
 ## Use the JavaScript Console to Check the Value of a Variable
 
-This is a stub. <a href='https://github.com/freecodecamp/guides/tree/master/src/pages/certifications/javascript-algorithms-and-data-structures/debugging/use-the-javascript-console-to-check-the-value-of-a-variable/index.md' target='_blank' rel='nofollow'>Help our community expand it</a>.
+In order to open up the Javascript Console of your Browser (chrome,firefox,safari,opera,edge) press: [here](https://guide.freecodecamp.org/certifications/javascript-algorithms-and-data-structures/debugging/understanding-the-differences-between-the-freecodecamp-and-browser-console)
```
As mentioned by @RandellDawson, there shouldnt be a reference to the challenge which the camper hasnt encountered yet (atleast for this one).

You can instead write couple of instructions?

AlexandreLeFou replied:
Hello @ezioda004  yeah sure I can write something extra no problem but if you want to specify regarding what would you like me to write one extra paragraph etc?

RandellDawson replied:
@AlexandreLeFou Yes, just make it as concise as possible.  Looking forward to your addition!

thecodingaviator replied:
@AlexandreLeFou Any progress?

ezioda004 reviewed (changes requested):
Please make the changes as mentioned below.

ezioda004 reviewed (commented):
Hi @AlexandreLeFou,
I'm closing this PR as its gone stale, please feel free to make a new PR in the future.

Happy contributing!

AlexandreLeFou commented:
Really sorry I shall try and see the bug in the weekend+ remake a new pr :)

Get Outlook for Android<https://aka.ms/ghei36>

________________________________
From: Aditya <notifications@github.com>
Sent: Wednesday, March 6, 2019 9:45:27 PM
To: freeCodeCamp/freeCodeCamp
Cc: AlexandreLeFou; Mention
Subject: Re: [freeCodeCamp/freeCodeCamp] fixed stub/added solution/+link to open terminal (#34252)


@ezioda004 commented on this pull request.

Hi @AlexandreLeFou<https://github.com/AlexandreLeFou>,
I'm closing this PR as its gone stale, please feel free to make a new PR in the future.

Happy contributing!

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://github.com/freeCodeCamp/freeCodeCamp/pull/34252#pullrequestreview-211433805>, or mute the thread<https://github.com/notifications/unsubscribe-auth/Akez6Tr6TAcizbccS6qrtBANPaKmm5D3ks5vUBrXgaJpZM4YTlXs>.
